src/hotspot/share/gc/shenandoah/shenandoahVMOperations.hpp

```diff
@@ -38,16 +38,21 @@ class ShenandoahFullGC;
 //   - VM_ShenandoahFinalMarkStartEvac: finish up concurrent marking, and start evacuation
 //   - VM_ShenandoahInitUpdateRefs: initiate update references
 //   - VM_ShenandoahFinalUpdateRefs: finish up update references
+//   - VM_ShenandoahFinalRoots
```

Comment:
If we add it here, let's provide a comment:

```
// - VM_ShenandoahFinalRoots: finish up the roots, shortcut cycle
```

Reply:
https://bugs.openjdk.org/browse/JDK-8344151